src/Services/PreHashStrings/LegacyPreHashString.php

```diff
@@ -162,6 +162,11 @@ public static function supportsService(string $service): bool
         return in_array($service, static::SUPPORTED_SERVICES);
     }
 
+    /**
+     * Validate and normalise the security and request packets.  Currently
+     * nothing is done with the request packet, but future pre-hash schemes
+     * may require it.
+     */
```

Comment:
Might be worth adding the first part to the interface, too, if it's not already there.

Reply:
Yep, it's already there.